tdrs-backend/tdpservice/email/email.py

```diff
@@ -78,6 +77,6 @@ def filter_valid_emails(emails, logger_context=None):
                 logger_context=logger_context
             )
     if len(valid_emails) == 0:
-        raise ValidationError("No valid emails provided.")
+        logger.warn("No valid emails provided.")
```

Comment:
can't we add the logger_context? 

I am not sure if this log will help much in debugging.

Reply:
Should we also send a log to the DAC just to have extra coverage?

Reply:
I don't think that helps. Adding the context might have helped resolve the issue, however if we don't submit the context then this log is meaningless since it is not obvious why we were trying to send this email

Reply:
Take a look at what I changed. Logging to the DAC includes the context